src/renderer/components/TimelineSpace/SideMenu.vue

```diff
@@ -199,6 +202,15 @@ export default {
           color: #409eff;
         }
       }
+    }
+
+    .profile-wide {
+      color: #ffffff;
```

Comment:
__[stylelint]__ <sub>posted by Sider</sub>

Expected "#ffffff" to be "#fff" (color-hex-length)